java/cli/src/main/java/org/ray/cli/RayCli.java

```diff
@@ -0,0 +1,326 @@
+package org.ray.cli;
+
+import com.beust.jcommander.JCommander;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.ArrayList;
+import net.lingala.zip4j.core.ZipFile;
+import net.lingala.zip4j.exception.ZipException;
+import org.ray.api.UniqueID;
+import org.ray.cli.CommandStart;
+import org.ray.cli.CommandStop;
+import org.ray.core.RayRuntime;
+import org.ray.core.model.RunMode;
+import org.ray.core.model.RayParameters;
+import org.ray.runner.RunInfo;
+import org.ray.runner.RunManager;
+import org.ray.runner.worker.DefaultDriver;
+import org.ray.spi.KeyValueStoreLink;
+import org.ray.spi.PathConfig;
+import org.ray.spi.RemoteFunctionManager;
+import org.ray.spi.StateStoreProxy;
+import org.ray.spi.impl.NativeRemoteFunctionManager;
+import org.ray.spi.impl.RedisClient;
+import org.ray.spi.impl.StateStoreProxyImpl;
+import org.ray.util.NetworkUtil;
+import org.ray.util.config.ConfigReader;
+import org.ray.util.logger.RayLog;
+import org.ray.util.FileUtil;
+
+/**
+ * Ray command line interface
+ */
+public class RayCli {
+
+  private static RayCliArgs rayArgs = new RayCliArgs();
+
+  private static RunManager startRayHead(RayParameters params, PathConfig paths,
+      ConfigReader configReader) {
+    RunManager manager = new RunManager(params, paths, configReader);
+
+    try {
+      manager.startRayHead();
+    } catch (Exception e) {
+      e.printStackTrace();
+      RayLog.core.error("error at RayCli startRayHead", e);
+      throw new RuntimeException("Ray head node start failed, err = " + e.getMessage());
+    }
+
+    RayLog.core.info("Started Ray head node. Redis address: " + manager.info().redisAddress);
+    return manager;
+  }
+
+  private static RunManager startRayNode(RayParameters params, PathConfig paths,
+      ConfigReader configReader) {
+    RunManager manager = new RunManager(params, paths, configReader);
+
+    try {
+      manager.startRayNode();
+    } catch (Exception e) {
+      e.printStackTrace();
+      RayLog.core.error("error at RayCli startRayNode", e);
+      throw new RuntimeException("Ray work node start failed, err = " + e.getMessage());
+    }
+
+    RayLog.core.info("Started Ray work node.");
+    return manager;
+  }
+
+  private static RunManager startProcess(CommandStart cmdStart, ConfigReader config) {
+    PathConfig paths = new PathConfig(config);
+    RayParameters params = new RayParameters(config);
+
+    // Get the node IP address if one is not provided.
+    //if (params.node_ip_address.length() == 0) {
+    //    params.node_ip_address = NetworkUtil.getIpAddress(null);
+    //}
```

Comment:
If the above code isn't needed, let's just remove it. otherwise it'll confuse people who read this.

Reply:
sure